### PR TITLE
added css-essentials article [OFWEB-982] 

### PR DESCRIPTION
**Pull request by SinisaG**
https://motainteam.atlassian.net/browse/OFWEB-982

@maximevaly - Can you please give me some feedback, if you would include something else to description? I tried to avoid repeating documentation, but not sure if it was successful. :) Do you think is too grid orientated?

### CODE REVIEW

File: _posts/2016-01-30-css-essentials.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+we come across a lot of different design challenges. As we all know, with complexity of design,
+complexity of CSS rises.
+
+In 2 years (it has just been 2 years of our first commit to webapp - yey!), we have wrote
+quite a few CSS tricks and decided to gather them in to one package.
+That is how [CSS-essentials] (https://github.com/Onefootball/css-essentials) was born.
+
+The main idea behind it is just publishing something that we find useful
+and want to share within our projects. Also we believe that exposing code to community
+can only make it better - two heads are better then one - and making your code public,
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: Unnecessary coma after "public"

File: _posts/2016-01-30-css-essentials.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+motivates you to write clearer, more readable code.  
+
+Currently, package contains our most used feature, which is our grid (and some mixins).
+We needed a grid which was 24 columns based and a lot of control over it (we have 6 different breakpoints)
+and decided to write our own. Adding a desire to experiment with modern css to the mix,
+our flex based, 24 columns grid was born. We use it in ["wild"] ((https://www.onefootball.com)) for over a year now
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: Broken link

File: _posts/2016-01-30-css-essentials.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+That is how [CSS-essentials] (https://github.com/Onefootball/css-essentials) was born.
+
+The main idea behind it is just publishing something that we find useful
+and want to share within our projects. Also we believe that exposing code to community
+can only make it better - two heads are better then one - and making your code public,
+motivates you to write clearer, more readable code.  
+
+Currently, package contains our most used feature, which is our grid (and some mixins).
+We needed a grid which was 24 columns based and a lot of control over it (we have 6 different breakpoints)
+and decided to write our own. Adding a desire to experiment with modern css to the mix,
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: Capitalize "CSS"

File: _posts/2016-01-30-css-essentials.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+In 2 years (it has just been 2 years of our first commit to webapp - yey!), we have wrote
+quite a few CSS tricks and decided to gather them in to one package.
+That is how [CSS-essentials] (https://github.com/Onefootball/css-essentials) was born.
+
+The main idea behind it is just publishing something that we find useful
+and want to share within our projects. Also we believe that exposing code to community
+can only make it better - two heads are better then one - and making your code public,
+motivates you to write clearer, more readable code.  
+
+Currently, package contains our most used feature, which is our grid (and some mixins).
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: "the" package

File: _posts/2016-01-30-css-essentials.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+We needed a grid which was 24 columns based and a lot of control over it (we have 6 different breakpoints)
+and decided to write our own. Adding a desire to experiment with modern css to the mix,
+our flex based, 24 columns grid was born. We use it in ["wild"] ((https://www.onefootball.com)) for over a year now
+and have not experienced any major problems.
+
+It is worth noting that our webapp tends to support browsers IE10+ and android browser 4.3+
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: I would emphasise a bit "to support **recent browsers** likes IE10+ and android browser 4.3+"

File: _posts/2016-01-30-css-essentials.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+(I list this two as bottom limitation, because..well..they are the special ones),
+so if your want support for older browsers, this might not be the best choice for you.
+
+However if you aim for new, modern browsers, then we encourage you to give it a try!
+We appreciate every open github issue, every pull request and every high-five!
+
+In future, we expect to update our CSS-essentials with more cool stuff, but will
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: In "the" future

File: _posts/2016-01-30-css-essentials.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+keep in mind to respect what the name suggests, and keep it lean and essential.
+
+You can view the usage demo [here] (http://5minfork.com/onefootball/css-essentials),
+just navigate to demo folder.
+
+Package is available through bower and npm.
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: I think we can skip the part after that.

**Comment by maximevaly** [comment]
Looks good, I'll give it another read tomorrow

**Comment by SinisaG** [comment]
Thanks, updated.

File: _posts/2016-01-30-css-essentials.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+tags: css, flex, onefootball, css-essentials, open source
+author_id: sinisag
+category: webapp
+---
+
+Working daily on our [onefootball webapp] (https://www.onefootball.com),
+we come across a lot of different design challenges. As we all know, with complexity of design,
+complexity of CSS rises.
+
+In 2 years (it has just been 2 years of our first commit to webapp - yey!), we have wrote
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: "we have written"

File: _posts/2016-01-30-css-essentials.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+quite a few CSS tricks and decided to gather them in to one package.
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: "in to" -> "into"

File: _posts/2016-01-30-css-essentials.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+In the future, we expect to update our CSS-essentials with more cool stuff, but will
+keep in mind to respect what the name suggests, and keep it lean and essential.
+
+You can view the usage demo [here] (http://5minfork.com/onefootball/css-essentials),
+just navigate to demo folder.
+
+Package is available through bower and npm.
+
+More information(documentation) can be found on [github page] (https://github.com/Onefootball/css-essentials).
```
Comment (maximevaly) [inline _posts/2016-01-30-css-essentials.md]: Maybe "You can find more information as well as a complete documentation on the [github page](https://github.com/Onefootball/css-essentials) of the project".

**Comment by maximevaly** [comment]
5minfork is not working for now, I think we should keep that PR open for a few days until it's fixed.

**Comment by SinisaG** [comment]
Updated. I removed the demo link until/if 5minut fork is back again.

**Comment by maximevaly** [comment]
:+1: let's merge!

**Pull request merged**